frontend/src/state/reducers/apiReducer.ts

```diff
@@ -59,9 +60,20 @@ export const majorsReducer = (
         draft.isFetchingMajors = true;
         return draft;
       case getType(fetchMajorsSuccessAction):
-        const majors: Major[] = action.payload.majors;
+        const majors: Major[] = action.payload.majors.slice(1);
+        // TODO remove once concentrations are in the backend
+        const DEFAULT_CONCENTRATIONS: Concentrations = {
+          minOptions: 0,
+          maxOptions: 1,
+          concentrationOptions: [],
+        };
+        majors.forEach(
+          (major: any) => (major.concentrations = DEFAULT_CONCENTRATIONS)
+        );
         draft.isFetchingMajors = false;
         draft.majors = majors;
+        // TODO: remove this mock major :
+        draft.majors.push(majorWithConcentrationData);
```

Comment:
Can we remove this since Iman merged in her new searchneu schema PR?

Reply:
Yes indeed, good catch